internal/pkg/create/create_test.go

```diff
@@ -128,6 +128,24 @@ func Test_generateGitZipFileURL(t *testing.T) {
 				httpClientMock.On("Get", mock.Anything).Return(nil, fmt.Errorf("HTTPClient error"))
 			},
 		},
+		"Returns the zip URL with .git suffix removed": {
+			templateURL: "https://github.com/slack-samples/deno-starter-template.git",
```

Comment:
📣 This IMO is the case we ought consider as "correct" for the `--template` flag since it appears on the "HTTPS" tab of the `Code` dropdown from the sample page:

🔗 https://github.com/slack-samples/deno-starter-template
:tv: https://github.com/slack-samples/deno-starter-template.git

FWIW we do not use the `.git` prefix in sample names but might recommend using the entire URL ending with `.git` if this is encountered later... Or recommend changing the sample name altogether, though I understand this can cause hassle 😉